setup.py

```diff
@@ -13,6 +14,15 @@ def readme():
         return re.sub(reg, r'\1', f.read())
 
 
+try:
+    pkg_resources.get_distribution('pip>=18.1')
+    heatshrink_requirement = [
+        'cython',
+        'heatshrink @ git+https://github.com/yawor/pyheatshrink@py3fix',
```

Comment:
Would it make sense to add these to `extras_require`? I hope that there will be a new pypi release for heatshrink with your fix, so that we need no git paths here.

Reply:
Actually that's not my fix. It was done by a different GitHub user some time ago, but the PR still waits to be merged. I've only created this fork to be 100% sure it won't suddenly disappear over night.
The extras_require may be a good option, but pip 18.1 or newer is still required to install patched version directly from git until a new version is released. Another way would be to temporarily publish a patched version to pypi under a different package name.